drivers/NFSSR.py

```diff
@@ -273,6 +274,7 @@ def generate_config(self, sr_uuid, vdi_uuid):
         resp['sr_uuid'] = sr_uuid
         resp['vdi_uuid'] = vdi_uuid
         resp['sr_sm_config'] = self.sr.sm_config
+        resp['sr_other_config'] = self.sr.other_config
```

Comment:
We usually take care of those aspects when we convert a PR to a patch in the patchqueue. So how should this be tested? With the patchqueue applied or not?